app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
```

Comment:
string: length - 1-255, default - 65536
text: length - 1-4294967296, default - 255
So it's necessary to validates length.